torch/export/dynamic_shapes.py

```diff
@@ -1191,3 +1193,280 @@ def apply_fixes(path, dim, dummy):
         return dim  # unchanged dim
 
     return _tree_map_with_path(apply_fixes, dynamic_shapes, dynamic_shapes)
+
+
+def _postprocess_serialized_shapes(dynamic_shapes, dims):
+    """
+    Sorts dims and returns dictionary format
+    """
+    from torch.utils._sympy.numbers import int_oo
+
+    dims = {
+        k: {
```

Comment:
nit: maybe we can define this as a dataclass above, and then use dataclasses.asdict to dump it into a dictionary?